css/index.css

```diff
@@ -1,3 +1,222 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+  margin: 0 auto;
+  border: 1px solid green;
+  padding: 20px 1.25%;
+}
+
+@media(max-width: 800px) {
+    img.logo {
+        float:center;
+        align-content: middle;
+        position: relative;
+    }
+     img.introimg {
+        display: none;
+    }
+    
+}
+
+@media(max-width: 500px) {
+    header {
+        align: center;
+    }
+    nav {
+        border: 1px solid gray;
+    }
+  .container header nav {
+    display: flex;
+    flex-direction: column;
+      float: center;
+      border: 1px solid gray;
+  }
+
+  .container header nav a {
+    margin-bottom: 20px;
+  }
+    
+    img.introimg {
+        display: none;
+    }
+    .top-content {
+        margin: 20px;
+        width: 100%;
+    }
+}
+header {
+	/* border: 5px dashed orange; */
+	padding: 20px 20px 20px;
+
+}
+
+nav {
+	/* border: 3px solid red; */
+	display: inline-block;
```

Comment:
No more inline-block!!!! let's forget about the ancient times and use display-box instead :P